show-me-the-visualization/README.md

```diff
@@ -232,38 +232,35 @@ Great, that looks very similar to the output we got from the command line.
 
 ## Vega-Lite specification
 
-There [many ways to do visualizations](https://github.com/sorrycc/awesome-javascript#data-visualization) on the web. My personal favorite at the moment is [Vega-Lite](https://vega.github.io/vega-lite/), so we will use it here.
-Vega-Lite is a JavaScript library which describes a plot using a JSON document. In Vega-Lite the JSON Document is called a specification and can be rendered to an interactive visualization.
+There [many ways to do visualizations](https://github.com/sorrycc/awesome-javascript#data-visualization) on the web. One
+of our favorites is [Vega-Lite](https://vega.github.io/vega-lite/), a JavaScript library which describes a plot using a
+JSON document called a _specification_.
 
-The root finding algorithm tries to find the x where y is zero.
-So let's plot the iteration index against the y found in each iteration to see how quickly it converged to an answer.
+The root finding algorithm tries to find the `x` where `y` is zero using a series of iterations.
+Let's plot the iteration `index` against the `y` found in each iteration to see how quickly it converged to an answer.
 
-The Vega-Lite specification is constructed out of the following blocks
-
-* `"data"`, the iterations we want to plot as an array of iteration objects
-* `"encoding"`, which field should go on which axis
-* `"mark"`, for line plot use [line](https://vega.github.io/vega-lite/docs/line.html) marker
+The generic structure of our Vega-Lite specification looks like this:
```

Comment:
I see you replace the bullet list with pseudo code, use `<>` instead of `()` for more commonly used placeholder chars.

Reply:
done